internal/libdocker/builder.go

```diff
@@ -57,15 +63,26 @@ func (b *Builder) BuildClientImage(ctx context.Context, name string) (string, er
 	dir := b.config.Inventory.ClientDirectory(name)
 	_, branch := libhive.SplitClientName(name)
 	tag := fmt.Sprintf("hive/clients/%s:latest", name)
-	err := b.buildImage(ctx, dir, branch, tag)
+	err := b.buildImage(ctx, dir, "Dockerfile", branch, tag)
 	return tag, err
 }
 
 // BuildSimulatorImage builds a docker image of a simulator.
 func (b *Builder) BuildSimulatorImage(ctx context.Context, name string) (string, error) {
 	dir := b.config.Inventory.SimulatorDirectory(name)
+	buildContextPath := dir
+	buildDockerfile := "Dockerfile"
+	// build context dir of simulator can be overridden with "context.txt" file containing the desired build path
```

Comment:
What's the purpose of this feature?

Reply:
This is out of scope for this PR, will remove.

Reply:
In scope or not, I'm still curious what this is about :)

Reply:
So initially, I was working on this change on a fork of the `ethereum-optimism/hive` fork of this repo, where this change exists - see commit [here](https://github.com/ethereum-optimism/hive/commit/656d759a1f302425274389b712316a0eec7f5995).
From what I can tell, this allows the user to add a custom relative path to use for build context. It seems to be useful when you want to build a simulator from a `Dockerfile` in an arbitrary dir, or when you have a `Dockerfile.<module>` naming convention.